ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -55,41 +63,56 @@ const getpaymentMethods = (
   {
     name: I18n.t("wallet.methods.card.name"),
     description: I18n.t("wallet.methods.card.description"),
+    icon: CreditCard,
     onPress: props.navigateToAddCreditCard,
     status: "implemented",
     section: "credit_card"
   },
   {
-    name: I18n.t("wallet.methods.postepay.name"),
-    description: I18n.t("wallet.methods.postepay.description"),
-    onPress: props.navigateToAddCreditCard,
-    status: "implemented",
-    section: "credit_card"
+    name: I18n.t("wallet.methods.paypal.name"),
```

Comment:
Since now we can add all the payment methods from this screen, what you think about removing `AddDigitalMethodScreen` and the related route `ROUTES.WALLET_ADD_DIGITAL_PAYMENT_METHOD` and navigation action `navigateToWalletAddDigitalPaymentMethod`?